packages/twenty-front/src/modules/object-record/relation-picker/components/SingleEntitySelectMenuItemsWithSearch.tsx

```diff
@@ -49,21 +52,13 @@ export const SingleEntitySelectMenuItemsWithSearch = ({
       excludedRelationRecordIds,
     });
 
-  const showCreateButton = isDefined(onCreate);
-
-  let onCreateWithInput = undefined;
-
-  if (isDefined(onCreate)) {
```

Comment:
FYI I think we should have kept that https://github.com/twentyhq/twenty/issues/8529#issue-2665477804

Reply:
Sorry to have introduced that 😢
I think I didn't really understand what it was used for.
I was under the impression my change didn't impact the public API of the component... but it seems I was wrong, sorry.

It's weird that TS didn't catch it

Reply:
No problem at al! I should have seen it 😅 (+ lack of good tests on our codebase). That code looks ugly I would have been tempted to remove it too!